<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="VLay_Left">
        <item>
         <layout class="QHBoxLayout" name="HLay_Left_ToolBar">
          <item>
           <widget class="QToolButton" name="TBtn_L_Prev">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="previous" resource="resource.qrc">
              <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TBtn_L_Next">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="next" resource="resource.qrc">
              <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TBtn_L_Refresh">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="reload" resource="resource.qrc">
              <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineEdit_L_Url">
            <property name="placeholderText">
             <string>http://</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="WebView_Left">
          <property name="url">
           <url>
            <string>https://vk.com/pomodorka_zr</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="VLay_Right">
        <item>
         <layout class="QHBoxLayout" name="HLay_Right_ToolBar">
          <item>
           <widget class="QToolButton" name="TBtn_R_Prev">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="previous" resource="resource.qrc">
              <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TBtn_R_Next">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="next" resource="resource.qrc">
              <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="TBtn_R_Refresh">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset theme="reload" resource="resource.qrc">
              <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineEdit_R_Url">
            <property name="placeholderText">
             <string>http://</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="WebView_Right">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>TBtn_L_Prev</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Left</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TBtn_L_Next</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Left</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TBtn_L_Refresh</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Left</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TBtn_R_Prev</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Right</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TBtn_R_Next</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Right</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TBtn_R_Refresh</sender>
   <signal>clicked()</signal>
   <receiver>WebView_Right</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
